#3d point which the end of the arm is currently occupiying

geometry_msgs/Point point
float32 attack_angle
